pyvelox/fuzzer.cpp

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "fuzzer.h" // @manual
+#include "velox/functions/FunctionRegistry.h"
+#include "velox/functions/prestosql/registration/RegistrationFunctions.h"
+#include "velox/functions/sparksql/Register.h"
+
+namespace facebook::velox::py {
+using namespace velox;
+namespace py = pybind11;
+
+FunctionSignatureMap getPrestoSignatures() {
+  clearFunctionRegistry();
+  facebook::velox::functions::prestosql::registerAllScalarFunctions();
```

Comment:
should we expose the registry as a separate python function to make the behavior closer to Velox (instead of implicitly registering them)?